src/main/java/org/rumbledb/expressions/control/SwitchCase.java

```diff
@@ -21,33 +21,32 @@
 package org.rumbledb.expressions.control;
 
 
-import org.rumbledb.exceptions.ExceptionMetadata;
-import org.rumbledb.expressions.AbstractNodeVisitor;
 import org.rumbledb.expressions.Expression;
-import org.rumbledb.expressions.Node;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class SwitchCaseExpression extends Expression {
+/**
+ * This is a helper class that organizes the children expressions of a child expression.
+ * From a tree perspective, all expressions in there are considered
```

Comment:
For added clarity, could you spell out what 'there' means in the context. ie. 'in this class' or 'in conditionExpressions' etc.

Reply:
DONE.